api/package.json

```diff
@@ -142,19 +142,19 @@
 		"node-machine-id": "1.1.12",
 		"nodemailer": "6.9.1",
 		"object-hash": "3.0.0",
-		"openapi3-ts": "3.1.2",
+		"openapi3-ts": "3.2.0",
 		"openid-client": "5.4.0",
 		"ora": "5.4.1",
 		"otplib": "12.0.1",
-		"pino": "8.10.0",
+		"pino": "8.11.0",
 		"pino-http": "8.3.3",
 		"pino-http-print": "3.1.0",
-		"pino-pretty": "9.2.0",
-		"qs": "6.11.0",
+		"pino-pretty": "10.0.0",
+		"qs": "6.11.1",
 		"rate-limiter-flexible": "2.4.1",
-		"rollup": "3.15.0",
-		"samlify": "2.8.9",
-		"sanitize-html": "2.9.0",
+		"rollup": "3.19.1",
```

Comment:
Hmm I don't think this will have any serious impact on our usage, so I would do the update for now. If there really should be issues with this, we can downgrade rollup without any problems.

Reply:
Sounds good to me! 👍